teachopencadd/talktorials/T014_binding_site_detection/talktorial.ipynb

```diff
@@ -0,0 +1,2116 @@
+{
```

Comment:
<p>Maybe <code>pocket_id</code>  instead of <code>poc_id</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/86/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='75'/>

Reply:
<p>Changed to new_pocket_id</p>